lib/browser/api/web-contents.js

```diff
@@ -288,6 +288,13 @@ WebContents.prototype.getZoomFactor = function (callback) {
   })
 }
 
+WebContents.prototype.findInPage = function (text, options = {}) {
+  //TODO (nitsakh): Remove in 5.0
+  if (options.wordStart || options.medialCapitalAtWordStart)
```

Comment:
```suggestion
  if (options.wordStart != null || options.medialCapitalAtWordStart != null)
```

this will catch potential usages where the app passes `false` in some cases (e.g. when developer is testing) but `true` in other cases (e.g. when user flips some option)